packages/test-renderer/src/__tests__/__snapshots__/RTTR.core.test.tsx.snap

```diff
@@ -304,8 +304,8 @@ Array [
             "children": Array [],
             "props": Object {
               "args": Array [
-                2,
-                2,
+                6,
+                6,
```

Comment:
so this should be `2` the test renders:

```jsx
      <group>
        <mesh key="d" rotation-x={1}>
          <boxGeometry args={[6, 6]} />
          <meshBasicMaterial />
        </mesh>
        <mesh key="b" position-y={12}>
          <boxGeometry args={[4, 4]} />
          <meshBasicMaterial />
        </mesh>
        <mesh key="c" position-x={12}>
          <boxGeometry args={[2, 2]} />
          <meshBasicMaterial />
        </mesh>
      </group>
```

It's like the `args` aren't being reflected as updated?

Reply:
Snapshot is unchanged now.